examples/standalone-with-org-policies/main.tf

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
do we want a tf.vars example file similar to the other `standalone` example?

Reply:
Yes, the tv.vars should be similar in both. The difference in the example with org policies wil be in case the user wants to create the projects from scratch instead of using existent ones. In this case the project_ids need to be added in the tf.vars.

Reply:
updated file looks good, so resolving